### ISSUE TITLE
Manual checking accessability 

### ISSUE THREAD

**Issue by a20behta** [HGustavs/LenaSYS#11794]
Related to issue #9800 
The accessability program 
[lighthous](https://developers.google.com/web/tools/lighthouse) 
was used to find multible accessability issues. 
there were some things which needed to be checked manually if issues exist. 

picture below of the sugested things to check 

![image](https://user-images.githubusercontent.com/102468604/165065903-1b30eb80-12a0-46a0-9341-1161fe8cf898.png)

check if there is a issue. 
explain where it can be found on the lenasys website.

**Comment by b20hanwa** [comment]
**This comment applies to the first manual check on the list:**
<img width="326" alt="Accessability8" src="https://user-images.githubusercontent.com/81630271/165745010-1b2302df-2b7d-42ca-8919-32852ca733c9.png">

When tabbing through LenaSYS I came to understand that a lot of the interactive buttons on the page are not accessible via keyboard navigation. The buttons marked with red in the images below have this problem:


<img width="960" alt="Accessability1" src="https://user-images.githubusercontent.com/81630271/165745773-372fd774-36b6-4135-87a7-86cff5d09087.png">


<img width="960" alt="Accessability2" src="https://user-images.githubusercontent.com/81630271/165745796-b627b262-ddd5-468e-aa33-006dc52c45d3.png">


<img width="949" alt="Accessability3" src="https://user-images.githubusercontent.com/81630271/165745819-10db942b-c4ff-4791-aa9a-a15ddfb0510c.png">

When having a pop-up on the page, you first have to tab through the page in the background to reach the pop-up. The page in focus (in this case; the pop-up) should be the one that you can tab through. 


<img width="948" alt="Accessability4" src="https://user-images.githubusercontent.com/81630271/165745909-d484008f-84bb-4313-86c4-518a2ff344a3.png">


<img width="949" alt="Accessability5" src="https://user-images.githubusercontent.com/81630271/165746000-43cbd812-f21d-48a8-af01-1c3c1cc8c384.png">


<img width="963" alt="Accessability6" src="https://user-images.githubusercontent.com/81630271/165746058-b691e596-5fdf-4d59-be2a-ea900d5b3d06.png">


<img width="950" alt="Accessability7" src="https://user-images.githubusercontent.com/81630271/165746118-3f6d13e6-74e2-4214-9969-180f348e947a.png">

**Comment by b20hanwa** [comment]
This is what Lighthouse suggest as solution for the tab order on a page:

<img width="697" alt="Accessability9" src="https://user-images.githubusercontent.com/81630271/165749597-2458ec3c-4852-4238-a68a-da8796043f73.png">

**Comment by a20caros** [comment]
<img width="408" alt="image" src="https://user-images.githubusercontent.com/81676918/167568410-b82341f9-c4dc-4078-aac0-79ef2e7f3946.png">

<img width="893" alt="image" src="https://user-images.githubusercontent.com/81676918/167568268-f6ac0d56-44b6-41be-8975-2fa7a2825361.png">
For the "structured data is valid" availability, no errors were detected

**Comment by a20caros** [comment]
Manual accessability check that's not done:
<img width="402" alt="image" src="https://user-images.githubusercontent.com/81676918/167571430-965e0193-6f50-4fcd-95a5-3dfdcd38e408.png">
<img width="396" alt="image" src="https://user-images.githubusercontent.com/81676918/167571792-1413ce2e-cb15-480d-87c6-fcef97b865c2.png">
<img width="393" alt="image" src="https://user-images.githubusercontent.com/81676918/167572337-a40a78bb-a613-4e8a-8c79-7d2a6f1eee25.png">
<img width="398" alt="image" src="https://user-images.githubusercontent.com/81676918/167572520-d7717bd5-c875-4972-a1ea-29fd62827373.png">
